certify-sunbird-farmer.properties

```diff
@@ -0,0 +1,165 @@
+              'credential_configurations_supported' : { \
+                 'FarmerCredential' : {\
+                    'format': 'ldp_vc',\
+                    'scope' : 'sunbird_rc_insurance_vc_ldp',\
+                    'cryptographic_binding_methods_supported': {'did:jwk'},\
+                    'credential_signing_alg_values_supported': {'Ed25519Signature2020'},\
+                    'proof_types_supported': {'jwt': {'proof_signing_alg_values_supported': {'RS256', 'PS256'}}},\
+                    'credential_definition': {\
+                      'type': {'VerifiableCredential','FarmerCredential'},\
+                      'credentialSubject': {\
+                            'name': {'display': {{'name': 'Name','locale': 'en'}}}, \
+                            'dateOfBirth': {'display': {{'name': 'Date of Birth','locale': 'en'}}},\
+                            'highestEducation': {'display': {{'name': 'Highest Education','locale': 'en'}}},\
+                            'maritalStatus': {'display': {{'name': 'Marital Status','locale': 'en'}}},\
+                            'typeOfHouse': {'display': {{'name': 'Type of House','locale': 'en'}}},\
+                            'numberOfDependents': {'display': {{'name': 'Number of Dependents','locale': 'en'}}},\
+                            'address': {'display': {{'name': 'Address','locale': 'en'}}},\
+                            'phoneNumber': {'display': {{'name': 'Phone Number','locale': 'en'}}},\
+                            'phoneNumberOwner': {'display': {{'name': 'Phone Number Owner','locale': 'en'}}},\
+                            'knowsLanguage': {'display': {{'name': 'Known Languages','locale': 'en'}}},\
+                            'netWorth': {'display': {{'name': 'Net Worth','locale': 'en'}}},\
+                            'hero': {'display': {{'name': 'Image URL','locale': 'en'}}},\
+                            'works': {'display': {{'name': 'Work Type','locale': 'en'}}},\
+                            'type': {'display': {{'name': 'Farming Types','locale': 'en'}}},\
+                            'landArea': {'display': {{'name': 'Land Area','locale': 'en'}}},\
+                            'landOwnershipType': {'display': {{'name': 'Land Ownership Type','locale': 'en'}}},\
+                            'primaryCropType': {'display': {{'name': 'Primary Crop Type','locale': 'en'}}},\
+                            'secondaryCropType': {'display': {{'name': 'Secondary Crop Type','locale': 'en'}}}\
+                     }},\
+                    'display': {{'name': 'Farmer', \
+                                  'locale': 'en', \
+                                  'logo': {'url': 'https://${mosip.api.public.host}/inji/veridonia-logo.png','alt_text': 'a square logo of a Veridonia'},\
+                                  'background_color': '#FDFAF9',\
+                                  'background_image': { 'uri': 'https://${mosip.api.public.host}/inji/veridonia-logo.png' }, \

```

Comment:
we have to use the agristack logo
https://raw.githubusercontent.com/aranaravi/kendo-ui-core/master/Agro%20Vertias%20Logo.svg

Reply:
please update for all version drafts